### PR TITLE
(DPS-2055) Automated link checking in docs

### PR DESCRIPTION
**Pull request by arturfracala**
# 🚀 Links

- [Jira](https://livechatinc.atlassian.net/browse/DPS-2055)

# 📓 Description

### CODE REVIEW

File: .lycheeignore [hunk 1]
```diff
@@ -0,0 +1,19 @@
+john.doe@mycompany.com
```
Comment (jakubsikora) [inline .lycheeignore:1]: Are this files related to us at all? Looks like they are coming from some example?

**Review by jakubsikora — APPROVED** [review]
Nice work. 1 question.

**Review by bstopyra — APPROVED** [review]
